packages/sample-app/__tests__/webpack-config.test.js

```diff
@@ -8,7 +8,23 @@ const configFile = require.resolve('../webpack.config.babel')
 
 test('it exports the development config', t => {
   const originalConfig = requireConfig()
+  const module = Object.assign({}, originalConfig.module, {
```

Comment:
Didn't really expect that here. It's surely not wrong, but it there a background story why you added this just now? (Just curious :) )

Reply:
Because I made a change in `packages/sample-app/webpack.config.babel.js`:

```diff
-  typescript(),
+  typescript({ configFileName: path.resolve(__dirname, './tsconfig.json') }),
```